src/hutil.ml

```diff
@@ -18,40 +16,25 @@ value commd_no_params conf =
 
 value link_to_referer conf =
   let referer = Util.get_referer conf in
+  let retour = Util.capitale (Util.transl conf "back") in
   if referer <> "" then
-    let fname = "left.png" in
-    let wid_hei =
-      match Util.image_size (Util.image_file_name fname) with
-      [ Some (wid, hei) ->
-          " width=\"" ^ string_of_int wid ^ "\" height=\"" ^
-          string_of_int hei ^ "\""
-      | None -> "" ]
-    in
-    sprintf "<a href=\"%s\"><img src=\"%s/%s\"%s alt=\"&lt;&lt;\" title=\"&lt;&lt;\"%s></a>\n"
-      referer (Util.image_prefix conf) fname wid_hei conf.xhs
+    sprintf "<a href=\"%s\"><span class=\"fa fa-arrow-left fa-lg\" title=\"%s\"></span></a>\n" referer retour
   else ""
 ;
 
 value gen_print_link_to_welcome f conf right_aligned =
   if conf.cancel_links then ()
   else do {
-    let fname = up_fname conf in
-    let wid_hei =
-      match Util.image_size (Util.image_file_name fname) with
-      [ Some (wid, hei) ->
-          " width=\"" ^ string_of_int wid ^ "\" height=\"" ^
-          string_of_int hei ^ "\""
-      | None -> "" ]
-    in
     if right_aligned then
-      Wserver.printf "<div style=\"float:%s\">\n" conf.right
+      Wserver.printf "<div class=\"btn-group float-%s mt-2\">\n" conf.right
```

Comment:
Yes classes are the good one that is float-right and float-left, but I can add that nobody verified rtl language (hebrew for us) and they are probably a lot of tiny things to adjust for those but we can't read them so it's not easy!!